pkg/genericapiserver/genericapiserver.go

```diff
@@ -489,17 +489,17 @@ func (s *GenericAPIServer) init(c *Config) {
 	s.Handler = handler
 
 	// After all wrapping is done, put a context filter around both handlers
-	if handler, err := api.NewRequestContextFilter(s.RequestContextMapper, s.Handler); err != nil {
+	handlertmp, err := api.NewRequestContextFilter(s.RequestContextMapper, s.Handler)
```

Comment:
This isn't really an improvement.  I'm not sure that handlertmp is more readable than the else block.  I'd prefer the variable name be 

```
var err error
handler, err = ...
```

rather than declaring a new variable.